server/src/main/java/org/elasticsearch/action/bulk/TransportBulkAction.java

```diff
@@ -336,6 +369,22 @@ protected void doInternalExecute(Task task, BulkRequest bulkRequest, String exec
         }
 
         // Step 3: create all the indices that are missing, if there are any missing. start the bulk after all the creates come back.
```

Comment:
I think this comment is misleading now since it precedes a method called `indexData`

Reply:
You think the comment is misleading, or the method name is misleading? The method does what the comment says. I could rename it `createMissingIndicesAndIndexData`.